CHANGELOG.md

```diff
@@ -21,6 +21,8 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 - Added necessary permissions for api-resource-collector so that the new
   [rule](https://github.com/ComplianceAsCode/content/pull/8511)
   `cluster_logging_operator_exist` can be evaluated properly.
+- Added `maunalRules` to `TailoredProfile` CRD, user can choose to add the rule
+  there so that those rules will show Manual as results.
```

Comment:
We should explicitly say that the remediations are not created in this case, both here and in the CRD docs.